rdkit/Dbase/UnitTestDbConnect.py

```diff
@@ -207,7 +207,7 @@ def test_InsertData(self):
 
     self.assertEqual(len(conn.GetColumnNames(table=newTblName)), 3)
     conn.AddColumn(newTblName, 'val3', 'int')
-    #     conn.Commit()
+    conn.Commit()
```

Comment:
yeah, I couldn't figure out why it was commented out, but I'm pretty sure that was your code.